### DISCUSSION TITLE
自带的英文输入法在chromium系或者chrome当中没有候选词

### DISCUSSION THREAD

**Discussion by um-nyu** [fcitx5-android/fcitx5-android#423] [Q&A | 问答] [answered]
其他APP中均正常
![20240122_084108](https://github.com/fcitx5-android/fcitx5-android/assets/81240333/0fffee53-e7ef-4765-8a66-f290577e0921)

**Comment by rocka** [comment, marked as answer]
chrome 的地址框设置了 [`TYPE_TEXT_FLAG_NO_SUGGESTIONS`](https://developer.android.com/reference/android/text/InputType#TYPE_TEXT_FLAG_NO_SUGGESTIONS) 故补全被禁用

**Reply by um-nyu** [reply]
我用了小企鹅的rime插件，在chrome上也能显示英语候选词。然后用自带输入法也能正常显示，某些chromium浏览器也能显示，我感觉是输入法的原因。不过这不重要了，我想问下用了rime插件后，怎么切换rime的输入方案？
现在我是把小企鹅的rime只留英语方案.倒也凑合能用
![IMG_2013](https://github.com/fcitx5-android/fcitx5-android/assets/81240333/9b7fed49-12c5-4a42-84d9-35d909f41c8a)

**Reply by rocka** [reply]
1. 有的输入法不遵守 EditorInfo ，不代表我们就要学着它们一样；同样并不是所有的输入框都会提示输入法关闭补全。因为 RIME 没有 [InputType](https://developer.android.com/reference/android/text/InputType) / [CapabilityFlag](https://github.com/fcitx/fcitx5/blob/7a9121cf2e55023435028d98ec1fb387952b9b8a/src/lib/fcitx-utils/capabilityflags.h#L21) 或类似的机制，它不支持根据输入框的 [EditorInfo](https://developer.android.com/reference/android/view/inputmethod/EditorInfo) 开关英文补全。
2. 先切换到 Rime 输入法，键盘左上角 <kbd>></kbd> 箭头展开工具栏，展开之后右上角 <kbd>···</kbd> 三点打开菜单。菜单中第二行第一个 <kbd><></kbd> 按钮即为方案选单。

**Reply by um-nyu** [reply]
谢谢解答
